roles/ceph-osd/tasks/main.yml

```diff
@@ -105,9 +105,17 @@
   when:
     - openstack_keys_tmp is defined
 
+- name: include set_osd_pool_default_pg_num.yml
+  include_tasks: set_osd_pool_default_pg_num.yml
+
 # Create the pools listed in openstack_pools
 - name: include openstack_config.yml
   include_tasks: openstack_config.yml
   when:
     - openstack_config
     - inventory_hostname == groups[osd_group_name] | last
+# Create RBD pools 
```

Comment:
please add an empty line before the comment

Reply:
Ok, I've applied the recommendations and rebased. @leseb @guits